ecoscape_layers/layers.py

```diff
@@ -94,46 +154,76 @@ def get_range_from_iucn(self, species_name, input_ranges_gdb, output_path):
         output_src = None
 
         # Reset the GDAL config option
-        gdal.SetConfigOption('OGR_ORGANIZE_POLYGONS', 'DEFAULT')
+        gdal.SetConfigOption("OGR_ORGANIZE_POLYGONS", "DEFAULT")
 
-    def get_range_from_ebird(self, species_code, output_path):
+    def get_range_from_ebird(self, species_code: str, output_path: str):
         """
         Gets range map in geopackage (.gpkg) format for a given bird species.
-
         :param species_code: 6-letter eBird code for a bird species.
         :param output_path: path to write the range map to.
         """
+
+        # check if the species code is a scientific name
+        if len(species_code) != 6:
```

Comment:
I removed all checks for 6 letter codes from this PR. I don't think there is a downside to removing this check because fetching data will fail if the wrong species code is put in.